package/MDAnalysis/analysis/helanal.py

```diff
@@ -129,41 +129,27 @@ def center(coordinates):
     """
     return np.mean(coordinates, axis=0)
 
-
-def veclength(v):
-    """Length of vector *v*."""
-    # note: this is 3 times faster than np.linalg.norm
-    return np.sqrt(np.dot(v, v))
-
-
-vecscaler = np.dot
-
+veclength = mdamath.norm
 
 def vecnorm(a):
     """Return a/|a|"""
     return a / veclength(a)
 
-
 def vecangle(a, b):
```

Comment:
mdamath.angle already covers nan issues, so this function can disappear (find and replace vecangle -> mdamath.angle)